src/tools/fs_utils.rs

```diff
@@ -162,10 +162,6 @@ fn main() {
             *metadata = FilesystemMetadata::blank_fs_init();
             drop(metadata);
             load_fs_special_files(&utilcage);
-
-            let metadata2 = FS_METADATA.read().unwrap();
-            persist_metadata(&*metadata2);
-            return;
```

Comment:
Wait why don't you want us to persist formatting the lindfs?

Reply:
It now falls through to rustfinalize which will persist.